library/component/toast/toast.css

```diff
@@ -0,0 +1,31 @@
+#toast{
+    min-width:25rem;
+    height:5rem;
+    display:none;
+    justify-content:flex-start;
+    align-items:center;
+    padding-left:1rem;
+    position:fixed;
+    background-color:var(--primary-bgcolor);
+    color:var(--primary-color);
+    font-size:1.8rem;
+    border:1px solid var(--primary-color);
+    border-radius:5px;
```

Comment:
use rem

Reply:
Plz correct me if I'm wrong I heard somewhere that to use rem if it is >= 10px (1 rem). So can i use rem for 5 px?

Reply:
tbh it's a personal choice, but rem enables resizing as per the user set font size so the UI does not mess up if the user has set font size anything other than what you had while developing.